main.tf

```diff
@@ -142,14 +159,32 @@ module "container_definition" {
       hostPort      = 0
       protocol      = "tcp"
     },
+    {
+      containerPort = 8000
+      hostPort      = 0
+      protocol      = "tcp"
+    },
```

Comment:
what is this used for? Should we use the gateway port instead?

Reply:
http entrypoint for traefik

Reply:
why not use one of the existing ones?